src/acl/class.test.ts

```diff
@@ -49,11 +50,18 @@ function addAclRuleQuads(
   type: "resource" | "default",
   agentClass:
     | "http://xmlns.com/foaf/0.1/Agent"
-    | "http://www.w3.org/ns/auth/acl#AuthenticatedAgent",
-  ruleIri?: IriString
+    | "http://www.w3.org/ns/auth/acl#AuthenticatedAgent"
+    | string,
+  ruleIri?: IriString,
+  targetType?:
+    | "http://www.w3.org/ns/auth/acl#agentClass"
+    | "http://www.w3.org/ns/auth/acl#agent"
+    | "http://www.w3.org/ns/auth/acl#agentGroup"
```

Comment:
No biggie (especially since it's just a helper function for a test), but you can specify a default value in JS as follows:

```suggestion
  targetType:
    | "http://www.w3.org/ns/auth/acl#agentClass"
    | "http://www.w3.org/ns/auth/acl#agent"
    | "http://www.w3.org/ns/auth/acl#agentGroup"
    = "http://www.w3.org/ns/auth/acl#agentClass"
```

(or whatever formatting Prettier would choose)